docscan/session/create/check/liveness.go

```diff
@@ -34,36 +45,35 @@ func NewRequestedLivenessCheckBuilder() *RequestedLivenessCheckBuilder {
 
 // RequestedLivenessCheckBuilder builds a RequestedLivenessCheck
 type RequestedLivenessCheckBuilder struct {
-	livenessType string
-	maxRetries   *int
-	//TODO: do we want this with pointer so we can set to nil?
+	livenessType livenessType
+	maxRetries   int
 }
 
 // ForZoomLiveness sets the liveness type to "ZOOM"
-func (builder *RequestedLivenessCheckBuilder) ForZoomLiveness() *RequestedLivenessCheckBuilder {
-	return builder.ForLivenessType(constants.Zoom)
+func (b *RequestedLivenessCheckBuilder) ForZoomLiveness() *RequestedLivenessCheckBuilder {
+	return b.ForLivenessType(zoom)
 }
 
 // ForLivenessType sets the liveness type on the builder
-func (builder *RequestedLivenessCheckBuilder) ForLivenessType(livenessType string) *RequestedLivenessCheckBuilder {
-	builder.livenessType = livenessType
-	return builder
+func (b *RequestedLivenessCheckBuilder) ForLivenessType(livenessType livenessType) *RequestedLivenessCheckBuilder {
```

Comment:
You sure you want this to be exported? There seems to be just one option.

If you do want it exported, you should make `livenessType` exported as well.

Reply:
These were added with strings originally so that the SDKs could take advantage of new liveness types without an update. Just realised if we make this an aliased type, it is no longer possible to do that. This is the case for all the other builders too.

I think I might return these to strings, but we can still keep them in the same places to aid separation of concerns. 
@davidgrayston would you agree that I'll need to keep these as strings?

Reply:
Other SDKs allow any string to be provided, so I think it would make sense to do the same here. e.g. https://github.com/getyoti/yoti-dotnet-sdk/blob/master/src/Yoti.Auth/DocScan/Session/Create/Check/RequestedLivenessCheckBuilder.cs#L15

Reply:
Yep, reverted back in 9c32f03

Reply:
> You sure you want this to be exported? There seems to be just one option.
> 
> If you do want it exported, you should make `livenessType` exported as well.

Sorry, never answered this - even though there's only one option now, we want to allow for future options to be added in the future, with the SDK being able to take advantage of it before we explicitly add a supported method